dotnet/src/SemanticKernel.Abstractions/Kernel.cs

```diff
@@ -152,6 +140,15 @@ public sealed class Kernel
         Interlocked.CompareExchange(ref this._promptFilters, [], null) ??
         this._promptFilters;
 
+    /// <summary>
+    /// Gets the collection of auto function invocation filters available through the kernel.
+    /// </summary>
+    [Experimental("SKEXP0001")]
+    public IList<IAutoFunctionInvocationFilter> AutoFunctionInvocationFilters =>
```

Comment:
Although `Auto` can be understood between us, this can be misunderstood between what is a Kernel Function Invocation and a `Function Calling Request Invocation` from the LLM, adding calling gives a bit more clarity to this.

```suggestion
    public IList<IAutoFunctionInvocationFilter> AutoFunctionCallFilters =>
```

Reply:
I would like to keep name of interface and kernel property aligned. So this change will require to rename interface as well. Interface name is based on `ToolCallBehavior.AutoInvokeKernelFunctions`. `Auto` in this case should point the difference between kernel function invocation and automatic function invocation. But I'm happy to discuss it further. @matthewbolanos could provide input here as well.